django-stubs/db/models/expressions.pyi

```diff
@@ -133,7 +133,7 @@ class F(Combinable):
         reuse: set[str] | None = ...,
         summarize: bool = ...,
         for_save: bool = ...,
-    ) -> F: ...
+    ) -> Ref: ...
```

Comment:
Since I'm planning to do a release soon, I'll go ahead and merge this with `-> Expression` for now, which is already an improvement.

If you believe this should be constrained further to `Ref`, please do open another pull request.

```suggestion
    ) -> Expression: ...
```